metals/src/main/scala/scala/meta/internal/metals/codeactions/RewriteBracesParensCodeAction.scala

```diff
@@ -92,4 +107,15 @@ class RewriteBracesParensCodeAction(
 object RewriteBracesParensCodeAction {
   val toParens = "Rewrite to parenthesis"
   val toBraces = "Rewrite to braces"
+  private val applyWithSingleFunction: Term.Apply => Boolean = {
```

Comment:
Can it be just a private method in the class? Any particular reason for moving it to the object?